robotidy/files.py

```diff
@@ -7,7 +7,7 @@
 from pathspec import PathSpec
 
 
-DEFAULT_EXCLUDES = r"/(\.direnv|\.eggs|\.git|\.hg|\.nox|\.tox|\.venv|venv|\.svn)/"
+DEFAULT_EXCLUDES = r"/(\.direnv|\.eggs|\.git|\.hg|\.nox|\.tox|\.venv|venv|\.svn)/|(.*\.pyi?$)"
```

Comment:
What if you edit and save `.json` file? So rather than specyfing what we want to include we should check if the file is in our includes - INCLUDE_EXT. See here how it's done https://github.com/MarketSquare/robotframework-tidy/blob/a1652d18ef8b385f09b71afcf4acf7d9f72d580b/robotidy/files.py#L133